packages/grid/test/tree-toggle.common.js

```diff
@@ -1,9 +1,6 @@
 import { expect } from '@esm-bundle/chai';
-import { click, fixtureSync } from '@vaadin/testing-helpers';
+import { click, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
```

Comment:
nit: `nextFrame` import is currently unused.

Reply:
Done